5-infrastructure/modules/base-env/boa-gke.tf

```diff
@@ -122,4 +124,40 @@ module "clusters" {
       node_metadata      = "GKE_METADATA_SERVER"
     }
   ]
+  node_pools_oauth_scopes = {
+    "all" : [
+      "https://www.googleapis.com/auth/cloud-platform",
+      "https://www.googleapis.com/auth/logging.write",
+      "https://www.googleapis.com/auth/monitoring",
+      "https://www.googleapis.com/auth/devstorage.read_only"
+    ],
+    "default-node-pool" : []
+  }
+}
+
+resource "google_binary_authorization_policy" "policy" {
+  count   = var.enforce_bin_auth_policy ? 1 : 0
+  project = var.boa_gke_project_id
+
+  global_policy_evaluation_mode = "ENABLE"
+  default_admission_rule {
+    evaluation_mode         = "REQUIRE_ATTESTATION"
+    enforcement_mode        = "DRYRUN_AUDIT_LOG_ONLY"
+    require_attestations_by = local.bin_auth_attestors
+  }
+  dynamic "admission_whitelist_patterns" {
+    for_each = local.allowlist_patterns
+    content {
+      name_pattern = admission_whitelist_patterns.value
+    }
+  }
+  dynamic "cluster_admission_rules" {
```

Comment:
Do we need cluster specific rules? I dont believe we are differentiating btw the policy enforced amongst the three clusters in an env?